### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by JDWNL**

### CODE REVIEW

File: docs/intro.mdx [hunk 1]
```diff
@@ -144,8 +148,8 @@ However the 3 core layers of Windmill, namely Scripts, Flows and Apps are comple
 		href="/docs/advanced/version_control"
 	/>
 	<DocCard
-		title="Deploy to Prod"
-		description="Deploy to prod using a staging workspace"
+		title="Deploy to Production"
```
Comment (HugoCasa) [inline docs/intro.mdx:151]: you should keep prod

**Review by HugoCasa — CHANGES_REQUESTED** [review]
very good

**Review by HugoCasa — APPROVED** [review]
it's fine now

**Review by hcourdent — COMMENTED** [review]
Good to me!

**Pull request merged**